client/landing/gutenboarding/onboarding-block/colors.scss

```diff
@@ -12,91 +12,91 @@
 		--highlightColor: var( --studio-blue-20 );
 	}
 
-	&[data-vertical*='football'] {
+	&[data-vertical='football' i] {
```

Comment:
I saw the discussion here (p1585642127142000-slack-luna) and I couldn't see the change of mind from partial-matches to full-matches (I mean removing the `*`), in fact, the opposite is mentioned. Does the `i` only work with full matches?

Also the `i` will fail in IE, not sure, but I suspect it might kill the whole CSS rule.

Reply:
Via other channel - we're fine with having IE gracefully degrade here 🙂